.github/workflows/test-platforms.yml

```diff
@@ -7,21 +7,63 @@ on:
     types: [opened, synchronize, reopened]
 
 jobs:
-  Build:
+  job:
+    name: ${{ matrix.os }} - ${{ github.workflow }}
     runs-on: ${{ matrix.os }}
```

Comment:
Should probably be declared after the srategy.matrix declaration

Reply:
Tried it. It failed.